core/src/test/java/org/elasticsearch/search/aggregations/bucket/FilterIT.java

```diff
@@ -145,6 +146,25 @@ public void testWithSubAggregation() throws Exception {
         assertThat((double) filter.getProperty("avg_value.value"), equalTo((double) sum / numTag1Docs));
     }
 
+    public void testAsSubAggregation() {
+        SearchResponse response = client().prepareSearch("idx")
+                .addAggregation(
+                        histogram("histo").field("value").interval(2l).subAggregation(
```

Comment:
Nit: `2L` is easier to read than `2l` (because in some fonts it looks like `21`). Same comment [here](https://github.com/elastic/elasticsearch/pull/15998/files#diff-0586a59911282a3e7b8f2a24cd5ed044R212). Sorry. :(